scripts/rpm-prepare.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set +ex
+
+echo "Reading ODO_VERSION, ODO_RELEASE and GIT_COMMIT env, if they are set"
+export ODO_VERSION=${ODO_VERSION:=1.0.0-test}
+export ODO_RELEASE=${ODO_RELEASE:=1}
+export GIT_COMMIT=${GIT_COMMIT:=`git rev-parse --short HEAD 2>/dev/null`}
+export ODO_RPM_VERSION=${ODO_VERSION//-}
+
```

Comment:
It would be nice to print the values of `ODO_VERSION`, `ODO_RELEASE`,  `GIT_COMMIT`  and `ODO_RPM_VERSION` here as verification that the correct one are used